pkgs/development/libraries/libtiff/default.nix

```diff
@@ -18,6 +18,10 @@ stdenv.mkDerivation rec {
     sha256 = "0d46bdvxdiv59lxnb0xz9ywm8arsr6xsapi5s6y6vnys2wjz6aax";
   };
 
+  cmakeFlags = if stdenv.isDarwin then [
+    "-DCMAKE_SKIP_BUILD_RPATH=OFF"
+  ] else null;
```

Comment:
```suggestion
  cmakeFlags = lib.optionals stdenv.isDarwin [
    "-DCMAKE_SKIP_BUILD_RPATH=OFF"
  ];
```

Reply:
I did this to avoid the rebuild on linux. lib.optionals will evaluate to an empty list rather than null on linux and cause a rebuild.

Reply:
I am personally not a big fan of such tricks but if it takes to long to rebuild on staging-next then I guess it it is fine.

Reply:
How about I subsequently issue a PR against staging to make the BUILD_RPATH unconditional? Then this check will only be in place until the next staging-next merge, and the mass rebuild will be "free" alongside other mass rebulid commits in staging

Reply:
I am not sure if this is worth the effort.